### PR TITLE
LPS-69016 Changes the way the component sets its value

### PR DESCRIPTION
**Pull request by fernandosouza**

### CODE REVIEW

**Comment by fernandosouza** [comment]
@diogo-lins please follow this commit (https://github.com/brunobasto/liferay-portal/pull/810/commits/39ec937c475284a8c3c22709c36901355b97c48d)

**Comment by fernandosouza** [comment]
@brunobasto pull updated

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js [hunk 1]
```diff
@@ -143,6 +148,18 @@ AUI.add(
 						return values;
 					},
 
+					openList: function() {
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js]: Wouldn't be better to rely on Soy to do this?

Comment (fernandosouza) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js]: You mean by rendering the component every time the user open the it?

**Comment by brunobasto** [comment]
@fernandosouza, you already sent it to QA, right? If so, can you close this PR?

Thx

**Comment by fernandosouza** [comment]
Yes, @brunobasto. I'm closing it.

**Pull request closed without merging**